testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -314,6 +314,75 @@ if(devices.real().deviceType === 'desktop') {
     });
 }
 
+QUnit.module('Accessibility', ()=> {
+
+    QUnit.test('list should have a selectByClick=true by default', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: true }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.ok(isSelectByClick);
+    });
+
+    QUnit.test('list should have to change the value of selectByClick during initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: false }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.notOk(isSelectByClick);
+    });
+
+    QUnit.test('list should have to change the value of selectByClick after updating', function(assert) {
```

Comment:
If module has name 'selectByClick option', we may name this test as follows
'option should be changed after initialization'
or 
'selectByClick option should be changed after initialization'

Reply:
Fixed